Library/Homebrew/cmd/doctor.rb

```diff
@@ -435,6 +443,7 @@ def check_access_usr_local
 
     You should probably change the ownership and permissions of /usr/local
     back to your user account.
+      sudo chown $(whoami):admin /usr/local
```

Comment:
That message comes from check inside `brew install` which is correct behavior. Though, we may improve the message, mentioning chown or something.